### PR TITLE
chore(consensus): Add info log when producing equivocation

### PR DESCRIPTION
**Pull request by dist1ll**
This will be helpful in case we get paged, and need to find out who produced the equivocating block.

### CODE REVIEW

File: rs/consensus/src/consensus/validator.rs [hunk 1]
```diff
@@ -1095,17 +1095,24 @@ impl Validator {
...
+                        signer: proposal.signature.signer,
+                        version: proposal.content.version().clone(),
+                        height: proposal.height(),
+                        subnet_id: self.replica_config.subnet_id,
+                        hash1: proposal.content.get_hash().clone(),
+                        signature1: proposal.signature.signature.clone(),
+                        hash2: CryptoHashOf::new(existing_metadata.content.hash().clone()),
+                        signature2: existing_metadata.signature.signature,
+                    };
+                    info!(
```
Comment (kpop-dfinity) [inline rs/consensus/src/consensus/validator.rs:1108]: maybe?
```suggestion
                    warn!(
```

File: rs/consensus/src/consensus/validator.rs [hunk 2]
```diff
@@ -1095,17 +1095,24 @@ impl Validator {
...
+                        self.log,
+                        "Equivocation found for node {:?}. Producing equivocation proof: {:?}",
+                        proof.signer,
```
Comment (kpop-dfinity) [inline rs/consensus/src/consensus/validator.rs:1111]: `signer` is part of `proof` already, so I guess it's not necessary to print it twice (unless it's more readable that way)?

```suggestion
                        self.log,
                        "Producing equivocation proof: {:?}",
```

**Review by kpop-dfinity — APPROVED** [review]

File: rs/consensus/src/consensus/validator.rs (hunk 2 above)
Comment (dist1ll) [reply rs/consensus/src/consensus/validator.rs]: done

File: rs/consensus/src/consensus/validator.rs (hunk 1 above)
Comment (dist1ll) [reply rs/consensus/src/consensus/validator.rs]: sure

**Pull request merged**